### DISCUSSION TITLE
ipad full screen is cropped

### DISCUSSION THREAD

**Discussion by martinorob** [blinksh/blink#1588] [Support] [answered]
Hi all,
blink 15.0.10
if i start a mosh session on my iphone and then start the same mosh session on my ipad, i have this kind of screen.
Is there a settings to resolve this behaviour? 

Thanks
M.

![F1FBF51A-1BCD-4C64-8FFF-532285D9C9AC](https://user-images.githubusercontent.com/6664936/193541673-dd79633d-1776-4517-a20a-d71129b73bd7.png)

**Comment by martinorob** [comment, marked as answer]
Solved:

in .tmux.conf 

set -g mouse on
new-session -n $HOST
set -g window-size largest 
setw -g aggressive-resize on

**Reply by carloscabanero** [reply]
Great trick, didn't know about this one!
